hw/ip/otp_ctrl/data/earlgrey_skus/emulation/BUILD

```diff
@@ -56,7 +54,7 @@ otp_json(
                 "CREATOR_SW_CFG_RNG_EN": otp_hex(CONST.HARDENED_FALSE),
                 "CREATOR_SW_CFG_JITTER_EN": otp_hex(CONST.MUBI4_FALSE),
                 "CREATOR_SW_CFG_RET_RAM_RESET_MASK": otp_hex(0x0),
-                "CREATOR_SW_CFG_MANUF_STATE": otp_hex(CONST.MANUF_STATE.SIVAL),
+                "CREATOR_SW_CFG_MANUF_STATE": otp_hex(0x0),
```

Comment:
basically the final cleanup is renaming the `sival` [rom_ext](https://cs.opensource.google/opentitan/opentitan/+/earlgrey_1.0.0:sw/device/silicon_creator/rom_ext/sival/) `emulation` and binding this image to it. since this ROM_EXT has the fake owner init. Then when #25106 is cherry picked this manuf_state binding will go to `CONST.MANUF_STATE.PERSONALIZED` (as all ROM_EXTs regardless of SKU will be bound to that manuf_state)

Reply:
If this merges before #25178, then I will update #25178 to change the 0x0 to `CONST.MANUF_STATE.PERSONALIZED`.